src/huggingface_hub/hf_file_system.py

```diff
@@ -119,24 +115,6 @@ def _repo_and_revision_exist(
                 self._repo_and_revision_exists_cache[(repo_type, repo_id, None)] = True, None
         return self._repo_and_revision_exists_cache[(repo_type, repo_id, revision)]
 
-    def exists(self, path, **kwargs):
```

Comment:
Let's keep it and call `info` with detail=False no ? Otherwise it's quite expensive

same for isdir and isfile